src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Figure implements Behaviour, CalcArea {
+    // Why can't I write 2 behavior in 1 interface(draw,getArea)?
```

Comment:
Using multiple interfaces allows for more flexibility in the code. It allows different classes to implement only the methods they need, rather than implementing a large number of methods that may not be relevant to them. 

Here it would be better to have two separate interfaces with their own method, rather than combining them into a single interface. This would allow classes implementing these interfaces to only implement the specific methods they need, and it would make the code easier to read and maintain.

Reply:
Technically, you can write 2 methods in one interface, but it wouldn't be a good object-oriented programming practice.

Reply:
Thank you